src/ir/auth_logic/ast_test.cc

```diff
@@ -0,0 +1,31 @@
+//-----------------------------------------------------------------------------
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//----------------------------------------------------------------------------
+
+#include "src/ir/auth_logic/ast.h"
+#include "src/common/testing/gtest.h"
+#include <iostream>
+#include <memory>
```

Comment:
This is a nit, but the inclusion order is wrong here. According to the Google style guide, the inclusion order should have the associated  header first (ast.h), then system headers (iostream and memory), then project headers (gtest.h).